src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -305,12 +305,22 @@ protected void Stop()
                 var sb = new StringBuilder();
                 for (int i = 0; i < launchParams.Args.Length; i++)
                 {
-                    sb.Append(PowerShellContext.QuoteEscapeString(launchParams.Args[i]));
-                    if (i < launchParams.Args.Length - 1)
+                    string arg = launchParams.Args[i];
+                    if (arg.StartsWith("-") || arg.StartsWith("'") || arg.StartsWith("\""))
```

Comment:
As you say @rkeithhill, there are [several characters](https://github.com/PowerShell/PowerShell/blob/dedddb6b59b8b01fadb4312bbddc7992a6cbbe0a/src/System.Management.Automation/engine/parser/tokenizer.cs#L4300-L4303) PowerShell accepts to begin a parameter.